package.json

```diff
@@ -77,7 +77,7 @@
     "test:e2e": "yarn workspace @okta/e2e test",
     "test:e2e:lang": "yarn workspace @okta/e2e test:lang",
     "test:testcafe-setup": "mkdir -p test-reports/testcafe && yarn start --mock mockDuo --open false",
-    "test:testcafe-run": "yarn test -t testcafe -c 2 chrome:headless --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
+    "test:testcafe-run": "yarn test -t testcafe -c 5 chrome:headless --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
```

Comment:
Just curious, any reason why a default concurrency wasn't added to the testcaferc.js config file?

Reply:
TLDR: developer experience

When concurrency is added to the `testcaferc.js` file it'll naively spin up `n` containers regardless of what other flags/args `testcafe` was invoked with, e.g., `-t`, `-T` (we have a vscode launch configs to "Debug with TestCafe" which will typically run one fixture). If we set the default concurrency on the config, it causes a disruptive "flash" as the browsers are opened.

I kept the concurrency here in the script to avoid the need to override with `-c1` when you're just running one test.